web/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "serve": "vue-cli-service serve",
+    "serve": "vue-cli-service serve --port 3001",

```

Comment:
I think this change shouldn't be in PR ;)

Reply:
Yeah my bad fixing this: wasn't sure what port vue uses by default, so I created the container exposing 3001 for it and had to add it there.

Reply:
I pushed the fix for this.